doc/conf.py

```diff
@@ -67,17 +58,28 @@
 ]
 
 myst_fence_as_directive = ["mermaid"]
+myst_enable_extensions = [
```

Comment:
Can see holoviews also have deflist, which can't hurt to also add here:

`myst_enable_extensions = ["colon_fence", "deflist"]`

https://myst-parser.readthedocs.io/en/latest/syntax/optional.html